src/server/routes/authRoutes.js

```diff
@@ -2,9 +2,15 @@ import express from 'express';
 import AuthController from '../controllers/AuthController';
 
 const authRouter = express.Router();
-authRouter.post('/api/v1/user/signup', [
+authRouter.post(
+  '/api/v1/user/signup',
   AuthController.createUser(),
-  AuthController.authenticateUser()
-]);
+);
+
+authRouter.post(
+  '/api/v1/user/signin',
+  AuthController.isAuthenticated(),
```

Comment:
The `isAuthenticated` controller mounted here suggests that the token is sent along in the request header.

Reply:
Oh... my mistake, will fix that right away